modules/standard/Types.chpl

```diff
@@ -389,13 +391,20 @@ proc isAssignable(type t)    param  return isAssignableType(t);
 pragma "no doc"
 proc isConstAssignable(type t) param  return isConstAssignableType(t);
 pragma "no doc"
+proc isNothing(type t)  param return isNothingType(t);
+
 proc isDefaultInitializable(type t) param return isDefaultInitializableType(t);
 
 // Set 2 - values.
 /*
 Returns `true` if the argument is a primitive type,
 as defined by the language specification, or a value of a primitive type.
 */
```

Comment:
this looks like the wrong documentation comment

Reply:
should i add pragma "no doc"
or delete a comment?

Reply:
We need a comment that documents the method `isNothing`. It can be based on the comment for `isNothingType` as a starting point.